cyences_app_for_splunk/default/data/ui/views/cs_linux_reports.xml

```diff
@@ -50,7 +56,10 @@
       </html>
```

Comment:
Let's bro change Linux Auditd Technology Add-On with Cyences Add-on for Splunk and link to Add-on.